torch/fx/experimental/proxy_tensor.py

```diff
@@ -392,7 +392,7 @@ def can_handle_tensor(x):
 
 class PythonKeyTracer(Tracer):
     def __init__(self):
-        super().__init__()
+        super().__init__(autowrap_modules=())
```

Comment:
Oh, this is a little bit worrisome... I'm not quite sure how to handle this. The issue is that FX autowraps `math.sqrt` (and other such functions) into a special lambda. Then, we have no idea how to lower it.

Reply:
Why does it wrap it into a special lambda? Can we have fx have its own impl that we redirect through that isn't a lambda? fx.math_sqrt or smth?

Reply:
yeah I think that would be reasonable/preferable.

Reply:
Actually, I'm not sure this is worrisome 🤔 It doesn't seem to be breaking anything.

Reply:
So I'm gonna land it if it passes tests :^)